go.mod

```diff
@@ -24,6 +24,7 @@ require (
 	github.com/ghodss/yaml v1.0.0
 	github.com/go-ole/go-ole v0.0.0-20170209151332-de8695c8edbf // indirect
 	github.com/go-resty/resty v0.0.0-20170925192930-9ac9c42358f7
```

Comment:
Could we update resty in `bonsai/client.go` too so we can remove that version?

Reply:
sure

Reply:
I decided I would simply remove resty as a dependency from the bonsai package, as it's not really adding any value, and its use hasn't expanded so much that it would be difficult to remove.